stable/magic-namespace/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: v1
 name: magic-namespace
-version: 0.1.0
+version: 0.1.1
 appVersion: 2.8.1
+home: https://www.mediawiki.org
```

Comment:
I'm stumped as to where you found this reference?

Reply:
Hi @timstoop , I checked from google. I found some home keys use the same url as code source, some use the real home page url. Any way, a home key with any value will make the ci testing OK. I think it's good to have one value for this. The maintainer can correct this later. What's your idea about this? Just leave it empty?

Reply:
No sorry, I'm not okay with this. Upstream for this chart is the Kubernetes Helm project, actually, according to the image. You can link to that. Let's make sure that information that's added is actually correct.

Reply:
Thanks @timstoop 
In the values.yaml: repository: gcr.io/kubernetes-helm/tiller
Should the value for home key be "https://www.gcr.io/kubernetes-helm/"?
This site is unvisitable in China for the firewall reasons.